webpack/assets/javascripts/react_app/components/common/forms/Actions.js

```diff
@@ -21,3 +21,17 @@ export default ({ onCancel, disabled = false, submitting = false }) => (
     </div>
   </div>
 );
+
+FormActions.propTypes = {
+  disabled: PropTypes.bool,
+  submitting: PropTypes.bool,
+  onCancel: PropTypes.func,
```

Comment:
shouldn't it be required?

Reply:
The `onCancel`? why?
I can render it with `noop`. IMHO funcs should never be required.